experimental-design/README.adoc

```diff
@@ -141,5 +141,70 @@ Some search engines as MaxQuant use the Fraction Group to perform better statist
 - Comment [Fraction Group]: Some Quantitative tools use the Fraction Group to know which fractions belong together. In MaxQuant the Fraction Group is called "Experiment".
 
 
+[[encoding-MSRun-technical-details]]
+== MSRun technical details properties
 
+We RECOMMEND to encode some of the technical parameters of the mass spectrometry experiment as Comments (https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[Check what is a Comment in SDRF]) including the following parameters:
 
+- Protein Modifications <<encoding-protein-modifications>>
+- Precursor and Fragment mass tolerances <<encoding-tolerances>>
+- Enzyme <<encoding-enzymes>>
+
+[[encoding-protein-modifications]]
+=== Protein Modifications
+
+Sample modifications (PTMs) are originated from multiple sources: **technical modifications**, **isotope labeling** or the most **biologically relevant** the wide variety of chemical modifications after translation. The most common and widely studied post translational modifications include phosphorylation and glycosylation. Many of these post-translational modifications are critical to the protein's function.
+
+The current specification RECOMMEND to provide Sample modifications including the Amino acid affected, if is Variable or Static (Fixed) and other properties such as mass shift and position (e.g. anywhere in the sequence).
+
+The RECOMMENDED name of the column for sample modification parameters is:
+
+  Commet [modification parameters]
+
+.. NOTE: The `modification parameters` is the name of the ontology term https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001055[MS:1001055]
+
+For each modification, we will capture different properties in a `key=value` pair structure including name, position, etc. All the possible features available for modification parameters:
+
+
+|===
+|Property |Key |Example | Required/Optional |Comment
+
+|Name of the modification| NM | NM=Acetylation | Required | Name of the modification, for custom modifications can be a name defined by the user.
+
+|Database Accession| AC | AC=UNIMOD:1 | Optional | Accession in an external database UNIMOD or PSI-MOD supported.
+
+|Chemical Formula  | CF | CF=H(2)C(2)O| Optional | This is the chemical formula of the added or removed atoms. For the formula composition please follow the guidelines from http://www.unimod.org/names.html[UNIMOD]
```

Comment:
would be good to specify how isotopes are read here so users don't have to look up the link

Reply:
Isotopes are a special case of modifications and should be discussed in a different thread Because they can be used for Quantification analysis?

Reply:
There are different types of modifications that can include isotopes.
Some are used for quantification, yes, which would include TMT, iTRAQ, SILAC, etc:
Here's an example of how it's defined in Unimod, but I'm not sure if you want to use the same way, because your initial format of the chemical composition was different:
"TMT6plex": H(20) C(8) 13C(4) N 15N O(2)

But besides that, there are also other mods that are not necessarily used for quantification, e.g. deamidation in the presence of heavy water, used for the identification of glycosylated peptides:
"Deamidated:18O(1)": H(-1) N(-1) 18O

Reply:
> There are different types of modifications that can include isotopes.
> Some are used for quantification, yes, which would include TMT, iTRAQ, SILAC, etc:
> Here's an example of how it's defined in Unimod, but I'm not sure if you want to use the same way, because your initial format of the chemical composition was different:
> "TMT6plex": H(20) C(8) 13C(4) N 15N O(2)

@StSchulze @trishorts My current proposal now aims to put the CF as Unimod defines. That included all your previous cases. My point about quantification is that we probably need to add more information like m/z values of the TMT® fragment ions to be quantified for 6plex and 10plex: 126.12773 127.12476 .. and that would be probably an extension of the current format. 

> 
> But besides that, there are also other mods that are not necessarily used for quantification, e.g. deamidation in the presence of heavy water, used for the identification of glycosylated peptides:
> "Deamidated:18O(1)": H(-1) N(-1) 18O

Agree, the current `FC` can be used for that.

Reply:
Agree that the isotope of the atom should be specified by the user, and the default isotope should be documented in the format, independently of quant. 
The ions used for quant will be listed in the diagnostic/reporter ions section with individual CF and isotopic constitution, so this should not require an extension :)
Some modifications add and remove atoms, or replace them. You can provide the combination as a single CF (e.g. `CF=NOH(-1)` or `CF=O(-1)18O(+1)`), which was our original implementation, but our users found that having the _atoms added_ and _atoms removed_ separated was more intuitive (e.g. `atoms_added=NO atoms_removed=H` or `atoms_added=O atoms_removed=18O`).

Reply:
I think, `CF` should be up to the software provider. In the current specification, we RECOMMEND using UNIMOD Guidelines for the Chemical Formula (http://www.unimod.org/fields.html). 

@mvaudel we can also add more intuitive keys for atoms_added `(AD)` and atoms_removed `(AR)`. We should keep it simple for now the CF.